src/main/resources/application.properties

```diff
@@ -122,6 +123,8 @@ http.client.connection.timeout=${HTTP_CLIENT_CONNECTION_TIMEOUT:30000}
 http.client.read.timeout=${HTTP_CLIENT_READ_TIMEOUT:60000}
 http.client.connection.drafts.create.timeout=${HTTP_CLIENT_CONNECTION_DRAFTS_CREATE_TIMEOUT:1000}
 http.client.connection.drafts.timeout=${HTTP_CLIENT_CONNECTION_DRAFTS_TIMEOUT:500}
+http.client.connection.callbacks.timeout=${HTTP_CLIENT_CONNECTION_CALLBACKS_TIMEOUT:300000}
+http.client.read.callbacks.timeout=${HTTP_CLIENT_READ_CALLBACKS_TIMEOUT:300000}
```

Comment:
this is interesting here. We always set on the library a fixed timeout of 5 min. Question, for my knowledge: imagine we configure a callback timeout of 40 secs. When the future that executes the callback times out after 40secs and returns, is the connection cancelled and released to the http connection pools or that will happen only after the 5 mins timeout is reached?

Reply:
also, for my knowledge, this means that even if the service configures 10 min callback timeout on the definitions, the library will ignore that and timeout after max 5 min anyway am I correct?

Reply:
First question: connection is release to the pool as soon as it is cancelled after the configured 40s custom timeout.
Second question: Yes the library will time out after 5 mins.